lib/BuildSystem/BuildSystem.cpp

```diff
@@ -3320,6 +3320,195 @@ class ArchiveTool : public Tool {
   }
 };
 
+#pragma mark - SharedLibraryTool implementation
+
+class SharedLibraryShellCommand : public ExternalCommand {
+
+  std::string swiftExecutable = "swiftc";
+#if defined(_WIN32)
```

Comment:
Please add a comment that these are only defaults (to make clear that the compile time check for host OS isn't preventing cross compilation).